docs/README.md

```diff
@@ -1,67 +1,63 @@
-as building blocks to declare pipelines.
-
-A custom resource is an extension of Kubernetes API which can create a custom
-[Kubernetes Object](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#understanding-kubernetes-objects).
-Once a custom resource is installed, users can create and access its objects
-with kubectl, just as they do for built-in resources like pods, deployments etc.
-These resources run on-cluster and are implemented by
-[Kubernetes Custom Resource Definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions).
-
-High level details of this design:
-
-- [Pipelines](pipelines.md) do not know what will trigger them, they can be
-  triggered by events or by manually creating [PipelineRuns](pipelineruns.md)
-- [Tasks](tasks.md) can exist and be invoked completely independently of
-  [Pipelines](pipelines.md); they are highly cohesive and loosely coupled
-- [Tasks](tasks.md) can depend on artifacts and parameters created by other
-  tasks.
-- [Tasks](tasks.md) can be invoked via [TaskRuns](taskruns.md)
-- [PipelineResources](resources.md) are the artifacts used as inputs and outputs
-  of Tasks.
-
-## Usage
-
-- [How do I create a new Pipeline?](pipelines.md)
-- [How do I make a Task?](tasks.md)
-- [How do I make Resources?](resources.md)
-- [How do I control auth?](auth.md)
-- [How do I run a Pipeline?](pipelineruns.md)
-- [How do I run a Task on its own?](taskruns.md)
-- [How do I get logs?](logs.md)
-
-## Learn more
-
-See the following reference topics for information about each of the build
-components:
-
-- [`Task`](tasks.md)
-- [`TaskRun`](taskruns.md)
-- [`Pipeline`](pipelines.md)
-- [`PipelineRun`](pipelineruns.md)
-- [`PipelineResource`](resources.md)
-
-Additional reference topics not related to a specific component:
-
-- [Labels](labels.md)
-- [Logs](logs.md)
-
-## Try it out
-
-- Follow along with [the tutorial](tutorial.md)
-- Look at
-  [the examples](https://github.com/tektoncd/pipeline/tree/master/examples)
-
-## Related info
-
-If you are interested in contributing to the Tekton Pipeline project, see the
-[Tekton Pipeline contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md).
+Tekton Pipelines is a custom Kubernetes extension that installs and runs on your Kubernetes cluster.
+It defines a set of Kubernetes [Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) that act as building blocks from which you can assemble custom CI/CD pipelines. Once installed,
+Tekton Pipelines becomes available via the Kubernetes CLI (kubectl) and via API calls, just
+like pods and other resources. Tekton is open-source and part of the [CD Foundation](https://cd.foundation/),
+a [Linux Foundation](https://www.linuxfoundation.org/projects/) project.
```

Comment:
We use the word "custom" several times in this paragraph but it doesn't add a ton of meaning except when it's the name of a specific Kubernetes type (Custom Resource).  Suggest dropping its usage.

Reply:
Agreed, I was being over-specific as a newbie to the space. Removed.